OutOfSchool/OutOfSchool.WebApi/Controllers/ChildController.cs

```diff
@@ -67,6 +68,29 @@ public async Task<IActionResult> GetById(long id)
             return Ok(await service.GetById(id).ConfigureAwait(false));
         }
 
+        /// <summary>
+        /// Get all children from the database by parent id.
+        /// </summary>
+        /// <param name="id">Id of Parent.</param>
+        /// <returns>List of all children.</returns>
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetByParentId(long id)
+        {
+            this.ValidateId(id, localizer);
+
+            var children = await service.GetAllByParent(id).ConfigureAwait(false);
```

Comment:
I think we need to implement checkout whether ParentId belongs to User, who called method.

Reply:
Dear @OlhaHoliak,
Thank you a lot for your advice. I think it is a good idea, so I have implemented it.
Please review.
Thank you a lot.